dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import {
+  InputGroup,
+  InputGroupText,
+  DatePicker,
+  isValidDate,
+  yyyyMMddFormat,
+  Button,
+} from "@patternfly/react-core";
+import "./index.css"
+import { formatDate } from "../../../utils/dateFormatter";
+
+function DatePickerWidget({
+  dataArray,
+  setPublicData,
+  controllerName,
+  setDateRange,
+}) {
+  const [fromDate, setFromDate] = useState(
+    formatDate(new Date(1990,10,4))
+  );
```

Comment:
Here in this piece of code we are just declaring and initializing fromDate.Whenever we declare a new state using useState ,we have to provide a initial value,and then whenever a user perform some user actions,we can change the value of that state.Usually the initial value provided is kind of a generic value(like an empty array,empty object,0 if we are considering integers),so I thought providing a old date as an initial value to fromDate will be a good Idea.

This initial value does not have much effect to the fromDate date picker.The from date Date picker will not start from 1990 if that’s the idea this piece of code is giving.

Reply:
> Whenever we declare a new state using useState, we have to provide a initial value

While that makes perfect sense, the [Patternfly example code disagrees](https://www.patternfly.org/v4/components/date-picker/react-demos/) with you (not that it's necessarily an authoritative source, but...).  😒

Given that you've omitted the `value` attribute from the `DatePicker` tag for the `fromDate`, I suspect it really doesn't matter (from the user's perspective) what we initialize `fromDate` to (because it will never be displayed).  However, I would recommend against setting it to something arbitrary (setting it to either undefined (by omitting the value) or to something like "the origin" (e.g., zero) would be better).  

Instead, I would recommend setting it to something meaningful (like "a month ago") and actually using the value that you set it to (by adding the `value` attribute to the corresponding `DatePicker` tag).